integration_tests/run_pyspark_from_build.sh

```diff
@@ -191,10 +191,14 @@ else
     ## Under cloud environment, overwrite the '--std_input_path' param to point to the distributed file path
     INPUT_PATH=${INPUT_PATH:-"$SCRIPTPATH"}
 
+    if [[ "${TEST_FILE_OR_DIR}" == "" ]]; then
+        TEST_FILE_OR_DIR="${LOCAL_ROOTDIR}/src/main/python"
+    fi
+
     RUN_TESTS_COMMAND=("$SCRIPTPATH"/runtests.py
-      --rootdir
-      "$LOCAL_ROOTDIR"
-      "$LOCAL_ROOTDIR"/src/main/python)
+      --rootdir "$LOCAL_ROOTDIR"
+      "${TEST_FILE_OR_DIR}"
```

Comment:
We will have to remove the quotes surrounding this variable otherwise it will surround everything with a single quote and it won't pick up more than one test. 
```
> export TEST_FILE_OR_DIR="$PWD/merged-branch/integration_tests/src/main/python/arithmetic_ops_test.py::test_multiplication_mixed $PWD/merged-branch/integration_tests/src/main/python/arithmetic_ops_test.py::test_mod_pmod_by_zero"
+ exec python /home/rjafri/dev/tomee/merged-branch/integration_tests/runtests.py --rootdir /home/rjafri/dev/tomee/merged-branch/integration_tests '/home/rjafri/dev/tomee/merged-branch/integration_tests/src/main/python/arithmetic_ops_test.py::test_multiplication_mixed /home/rjafri/dev/tomee/merged-branch/integration_tests/src/main/python/arithmetic_ops_test.py::test_mod_pmod_by_zero' -n 9 -v -rfE '' --std_input_path=/home/rjafri/dev/tomee/merged-branch/integration_tests/src/test/resources --color=yes '' --junitxml=TEST-pytest-1709868661407960875.xml

=============================================================================================================== no tests ran in 8.86s ================================================================================================================

```

Reply:
Reworked. Additional issue we actually have to make sure that the paths are from LOCAL_ROOT_DIR
The new ways is TESTS='module::method..."